Blogpost for Sprint 177

ncounter opened the pull request.

eduardoj commented:
This blog post looks very much to the [previous blog post entry](https://deploy-preview-478--openbuildservice.netlify.app/2024/08/06/introduction-request-index/):
- Same title
- Same first paragraph

It doesn't mention the previous blogpost, which could start a series of blog posts dealing with the same topic.

I would expect more difference between both blog posts. It looks like this blog post entry is introducing this same feature from scratch, when it should focus on what has been introduced recently...

ncounter commented:
> This blog post looks very much to the [previous blog post entry](https://deploy-preview-478--openbuildservice.netlify.app/2024/08/06/introduction-request-index/):
> 
>     * Same title
> 
>     * Same first paragraph
> 
> 
> It doesn't mention the previous blogpost, which could start a series of blog posts dealing with the same topic.
> 
> I would expect more difference between both blog posts. It looks like this blog post entry is introducing this same feature from scratch, when it should focus on what has been introduced recently...

Would you mind taking a look at it now? I addressed your feedback, thank you.
